openapi-spec/swagger.yaml

```diff
@@ -141,6 +141,26 @@ paths:
           description: Forbidden
         '500':
           $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{projectId}/availabilityZones':
+    parameters:
+      - $ref: '#/parameters/projectId'
+    get:
+      tags:
+        - Pool
+      description: Lists availabilityZones without repeating for all block storage pool.
+      responses:
+        '200':
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Pool'
```

Comment:
From the code I saw you return a string array, but here in the api specification you specify an array of Pool struct, which is not right. Please fix this bug.